hw_web_client.py

```diff
@@ -0,0 +1,42 @@
+from time import time
+
+import requests
+
+task1 = {
+    'status': 'active',
+    'expires': time() + 20,
+    'name': 'meow1',
+    'description': 'meow1',
+}
+task2 = {
+    'status': 'active',
+    'expires': time() + 25,
+    'name': 'meow2',
+    'description': 'meow2',
+}
+task3 = {
+    'status': 'active',
+    'expires': time() + 30,
+    'name': 'meow3',
+    'description': 'meow3',
+}
+
+
+if __name__ == '__main__':
+    resp = requests.post('http://localhost:5000/api/tasks', json=[task1, task2, task3])
```

Comment:
Like more professional way testing of your API, you could try using `pytest`.

Reply:
Thanks for advice!
I have been using Pycharm's debugger here, for testing. Then, switched to Postman.